tests/cases/fourslash/toggleMultilineComment2.ts

```diff
@@ -0,0 +1,35 @@
+// If selection is outside of a multiline comment then insert comment
+// instead of removing.
+
+//// let var1/* = 1;
+//// let var2 [|= 2;
+//// let var3 */= 3;|]
+////
+//// [|let var4/* = 1;
+//// let var5 |]= 2;
+//// let var6 */= 3;
+////
+//// [|let var7/* = 1;
+//// let var8 = 2;
+//// let var9 */= 3;|]
+////
+//// /*let va[|r10 = 1;*/
+//// let var11 = 2;
+//// /*let var12|] = 3;*/
+
+verify.toggleMultilineComment(
+    `let var1/* = 1;
+let var2 *//*= 2;

```

Comment:
Yeah, this is one of those scenarios that are difficult to decide the best approach. 

IMO, the only reason to not merge the comments is because when we want to reverse the operation we might get a different result:. For example when using jsdoc:
```js
[|const foo = 1;

/** some documentation 
* more doc.
*/
function bar()  {}|]
```
this will result on:
```js
/*const foo = 1;

*//** some documentation 
* more doc.
*//*
function bar()  {}*/
```
Looks messy but in a way it respects existing comments. If you want to uncomment either `foo` or `bar` it should be ok. If we merge comments, the jsdoc syntax will be all mess up.

This is not only for jsdoc, pretty much any existing comment will experience this issue when uncommenting.